src/serialise/ahoXml/generate.ts

```diff
@@ -533,26 +479,39 @@ const mapAhoCXE01ToXml = (pncQuery: PncQueryResult): Cxe01 => ({
     : undefined
 })
 
-const mapAhoToXml = (aho: AnnotatedHearingOutcome): AhoXml => {
+const mapAhoToXml = (aho: AnnotatedHearingOutcome, validate = true): AhoXml => {
+  const xmlnsTags = {
+    "@_xmlns:br7": "http://schemas.cjse.gov.uk/datastandards/BR7/2007-12",
+    "@_xmlns:ds": "http://schemas.cjse.gov.uk/datastandards/2006-10",
+    "@_xmlns:xsi": "http://www.w3.org/2001/XMLSchema-instance"
+  }
+  const hearingOutcome = {
+    "br7:HearingOutcome": {
+      "br7:Hearing": mapAhoHearingToXml(aho.AnnotatedHearingOutcome.HearingOutcome.Hearing),
+      "br7:Case": mapAhoCaseToXml(aho.AnnotatedHearingOutcome.HearingOutcome.Case),
+      ...(!validate && xmlnsTags)
+    }
+  }
+
   return {
     "?xml": { "@_version": "1.0", "@_encoding": "UTF-8", "@_standalone": "yes" },
-    "br7:AnnotatedHearingOutcome": {
-      "br7:HearingOutcome": {
-        "br7:Hearing": mapAhoHearingToXml(aho.AnnotatedHearingOutcome.HearingOutcome.Hearing, aho.Exceptions),
-        "br7:Case": mapAhoCaseToXml(aho.AnnotatedHearingOutcome.HearingOutcome.Case, aho.Exceptions)
-      },
-      "br7:HasError": hasError(aho.Exceptions),
-      CXE01: aho.PncQuery ? mapAhoCXE01ToXml(aho.PncQuery) : undefined,
-      "br7:PNCQueryDate": aho.PncQueryDate ? toISODate(aho.PncQueryDate) : undefined,
-      "br7:PNCErrorMessage": optionalText(aho.PncErrorMessage),
-      "@_xmlns:br7": "http://schemas.cjse.gov.uk/datastandards/BR7/2007-12",
-      "@_xmlns:ds": "http://schemas.cjse.gov.uk/datastandards/2006-10",
-      "@_xmlns:xsi": "http://www.w3.org/2001/XMLSchema-instance"
-    }
+    ...(validate
+      ? {
+          "br7:AnnotatedHearingOutcome": {
+            ...hearingOutcome,
+            CXE01: aho.PncQuery ? mapAhoCXE01ToXml(aho.PncQuery) : undefined,
+            "br7:PNCQueryDate": aho.PncQueryDate ? toISODate(aho.PncQueryDate) : undefined,
+            "br7:PNCErrorMessage": optionalText(aho.PncErrorMessage),
+            ...xmlnsTags
+          }
+        }
+      : {
+          ...hearingOutcome
+        })
   } as AhoXml
```

Comment:
I think we should remove this manual casting so the compiler can be sure we're returning the right thing